nerfstudio/scripts/render.py

```diff
@@ -432,6 +455,11 @@ class BaseRender:
     """If true, checks line-of-sight occlusions when computing camera distance and rejects cameras not visible to each other"""
     camera_idx: Optional[int] = None
     """Index of the training camera to render."""
+    kill_flag: List[bool] = field(default_factory=lambda: [False])
```

Comment:
Can we not expose this in the config? it feels like it should be an invisible class `self._kill_flag` variable, then the kill() function sets that to true.

Reply:
Still a hack but one option is:
```python
# `tyro.conf.Suppress` hides this field from the generated `ns-render` CLI.
_kill_flag: tyro.conf.Suppress[List[bool]] = field(default_factory=lambda: [False])
```